module_utils/certificate/providers/certmonger.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from io import StringIO
+
+# pylint: disable=no-name-in-module
+from ansible.module_utils.six.moves.configparser import ConfigParser
+
+from .base import CertificateRequestBaseProvider
+
+
+class CertificateRequestCertmongerProvider(CertificateRequestBaseProvider):
+    """Certmonger provider for certificate Linux System Role."""
+
+    def __init__(self, *args, **kwargs):
+        super(CertificateRequestCertmongerProvider, self).__init__(*args, **kwargs)
+
+        self.requests_dir = "/var/lib/certmonger/requests"  # TODO: Allow to override?
```

Comment:
It's possible it could be different by distribution. I'm not aware that it is different anywhere else though, but the packager may have done it without letting me know (why would they?)

Reply:
I'll check on Debian-based systems just in case.

Reply:
Debian-based uses on same location.
TODO removed on 2aeb77af734f4e369b359a41d5d4086f762ee7eb